src/TuxArchitectStores.js

```diff
@@ -19,15 +19,15 @@ ArchitectChain.prototype.itNeeds = function (inputOrStore) {
   //if inputOrStore is array
   if (Array.isArray(inputOrStore)) {
     var inputOrStoreLength = inputOrStore.length;
-    //loop through inputOrStore array and push register id into waitForIds for each element in array
+    //concat each inputOrStore's __registerId__ into array
```

Comment:
Change comment to ...inputOrStore's __registerId__ or set of __registerId__'s...